symplyphysics/laws/condensed_matter/height_of_the_pn_transition_barrier.py

```diff
@@ -0,0 +1,51 @@
+from sympy import (Eq, solve, log)
+from sympy.physics.units import boltzmann
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output
+)
+
+# Description
+## The p-n junction has a potential barrier preventing the movement of charge carriers. If the concentration
+## of donor impurities is significantly higher than the concentration of acceptor impurities, then the law for
+## the height of the barrier takes the form below.
+
+## Law is: F = (kT/q) * ln(Nd * Na / n), where
+## F - height of the potential transition,
+## n is concentration intrinsic charge carriers,
+## q - charge of the electron
+## Nd, Na, n - concentrations of donors, acceptors and intrinsic charge carriers, respectively,
+## k - Boltzmann constant,
+## T - temperature.
+
+height_barrier = Symbol("height_barrier", units.voltage)
+
+concentration_donors = Symbol("concentration_donors", 1 / units.length**3)
+concentration_acceptors = Symbol("concentration_acceptors", 1 / units.length**3)
+concentration_intrinsic = Symbol("concentration_intrinsic", 1 / units.length**3)
```

Comment:
Use the same naming as in your recent law: `charge_carriers_concentration`. Apply the same naming to donors and acceptors.